src/main/events/library/open-game.ts

```diff
@@ -1,16 +1,45 @@
 import { gameRepository } from "@main/repository";
-
 import { registerEvent } from "../register-event";
 import { shell } from "electron";
+import { promisify } from 'node:util';
+import { logger } from "@main/services";
+import { exec as execCallback } from "node:child_process";
+
+const exec = promisify(execCallback);
+
+const escapeShellPath = (path: string) => {
+  if (process.platform === 'win32') {
+    return `"${path}"`;
+  } else {
+    return path.replace(/ /g, '\\ ').replace(/\(/g, '\\(').replace(/\)/g, '\\)');
+  }
+};
 
 const openGame = async (
   _event: Electron.IpcMainInvokeEvent,
   gameId: number,
   executablePath: string
 ) => {
-  await gameRepository.update({ id: gameId }, { executablePath });
+  try {
+    await gameRepository.update({ id: gameId }, { executablePath });
 
-  shell.openPath(executablePath);
+    if (process.platform === 'linux') {
+      const { stdout: lutrisPath } = await exec('which lutris');
+      if (lutrisPath.trim()) {
+        await exec(`chmod +x "${executablePath}"`);
```

Comment:
What's the point of having `chmod +x` in here? Give execution permission to a file that can be downloaded from the internet can lead to major security issues and as far as I can see we are doing it without even telling the user about it.

As we do not handle game installation we should not handle file permissioning.

Reply:
Like i said in the original PR:

"it uses the lutris runtime thats default to run the executable. sadly it needs to make the .exe execute permissions but don't see a way around that."

Lutris does not run games without making the .exe an executable. dont see a way around it unless we want to make it a popup for the user to select which .exe to make executable?

Or we can look into using some other tool then lutris

Reply:
Any new ideas with this @scrlkx @gylli251?
I fixed the conflict, the PR is now updated with main branch

Reply:
I've tried this branch but I couldn't make it work. For some reason Lutris is not doing anything. Maybe one of you guys can share a sucefull scenario with version details? Anyway, as I will try to figure it out as soon as I have time to look into ti again.